pkg/alerts/nrql_conditions.go

```diff
@@ -353,7 +355,15 @@ func (a *Alerts) GetNrqlConditionQuery(
 		"id":        conditionID,
 	}
 
-	if err := a.client.NerdGraphQuery(getNrqlConditionQuery, vars, &resp); err != nil {
+	req, err := a.client.NewNerdGraphRequest(getNrqlConditionQuery, vars, &resp)
```

Comment:
Has the signature changed here?

Reply:
Nope, it's just a different way of performing a request. `NerdGraphQuery` uses `NewNerdGraphRequest`, so in order to set the request's error response, we need to create the request in a more granular way. We did this in the alert policy resource as well 😎 

I also think it would be a bit cleaner to have an additional argument passed to  `NerdGraphQuery ` ( e.g. `errorResponse`) and then set the errorResponse within that method, but would affect all the things.